utils/update_cime_machine_config.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import argparse
+
+from mache.cime_machine_config import (
+    compare_machine_configs,
+    extract_supported_machines,
+)
+from mache.io import download_file
+from mache.machines import get_supported_machines
+
+
+def main():
+    """
+    Main function to download the XML file, get supported machines, and extract
+    them, then compare the machine configurations between the old and new XML.
+    """
+    parser = argparse.ArgumentParser(
+        description='Get and display the updates made to '
+        'config_supported_machines.xml',
+    )
+
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='version',
+        version='',
```

Comment:
@altheaden, maybe it would make sense to show the `mache` version here?
```suggestion
        version=mache.version.__version__,
```

Reply:
Oops, I forgot to mention that I had left it blank as a placeholder since I wasn't sure what to put. I'll go ahead and make it the mache version!